src/components/AssetInfo/TokenSideBarHolder.tsx

```diff
@@ -59,8 +59,8 @@ const TokenSideBarHolder = ({
       {showChangeBeneficiary && (
         <TokenSideBarField
           id="changebeneficiary"
-          title="Change Beneficiary"
-          label="Change"
+          title="Endorse Change of Beneficiary"
```

Comment:
I am not sure if they asked to change the title here.they just wanted to keep it as it was.